diagnostic.md

```diff
@@ -57,13 +66,19 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+const hideBody = function () {
+  let element = $('li').first();
+  while (element !== $('body')) {
+    element = element.parent();
+  }
+  element.hide(1000); // Hide over 1 second
+}
 <!-- your answer ends here -->
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+$('div').on('click', doStuff);
```

Comment:
0.5/1. `doStuff` has to be a function that has been defined. Or else, you need to spell out the function.